ExampleOutput/Commands.g.cs

```diff
@@ -2,30 +2,35 @@
 using System.CommandLine;
```

Comment:
Note that one of two files are generated: A file like this if a delegate has been assigned to the CLI root (via `CliRoot.Create`) and the NoCommand.g.cs file if no delegate has yet been assigned. The NoCommand file provides Intellisense for `CliRoot.Create`